pontoon/pretranslation/tasks.py

```diff
@@ -37,19 +37,31 @@ def pretranslate(self, project_pk, locales=None, entities=None):
     """
     project = Project.objects.get(pk=project_pk)
 
-    log.info(f"Fetching pretranslations for project {project.name} started")
+    if not project.pretranslation_enabled:
+        log.info(f"Pretranslation not enabled for project {project.name}")
+        return
 
     if locales:
         locales = project.locales.filter(pk__in=locales)
     else:
         locales = project.locales
 
     locales = (
-        locales.filter(project_locale__readonly=False)
+        locales.filter(
+            project_locale__readonly=False, project_locale__pretranslation_enabled=True
+        )
```

Comment:
Just to check: This means that all projects which currently use pretranslation would have it disabled by this PR, right?

Reply:
Nice catch, there might be 3rd party instances that already use the feature.